lib/time/time.go

```diff
@@ -71,8 +71,25 @@ var Module = &starlarkstruct.Module{
 // NowFunc is a function that generates the current time. Intentionally exported
 // so that it can be overridden, for example by applications that require their
 // Starlark scripts to be fully deterministic.
+//
+// Deprecated: use SetNow() to set per-thread clock.
```

Comment:
// Deprecated: avoid updating this global variable
// and instead use SetNow on each thread to set its clock function.